sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkConfiguration.java

```diff
@@ -420,6 +428,9 @@ public String toShortString() {
     if (sinkType != DEFAULT.sinkType) {
       sb.append(String.format("; sinkType:%s", sinkType));
     }
+    if (exportSummaryToBigQuery != DEFAULT.exportSummaryToBigQuery) {
```

Comment:
Why are we comparing with default values ?

Reply:
Like all the other configuration items,  when we print the configuration, we print their value only if they are set by the user (i.e. their value is not the default value)